### DISCUSSION TITLE
Getting error after adding a Hub

### DISCUSSION THREAD

**Discussion by codexbushi** [wevm/frog-archived#72] [Q&A] [answered]
It seems like I am getting an error after I try to add a hub for frame data verification.

The error I get from the Warpcast frame validator is:

```
postFakeCastFrameAction 400 - Failed to post fake frame action: Failed to post to frame. Error: Request failed with status code 500
```

This happens right after I add a hub of any kind, for example:

```
const app = new Frog({
  basePath: '/api',
  hub: {
    apiUrl: 'https://hub.pinata.cloud',
  },
})
```

My frame is live up at: https://meme-frame-frog.vercel.app/api

Thanks for any help on this!

**Comment by jxom** [comment]
What do your error logs in Vercel display?

**Reply by codexbushi** [reply]
Two errors in the logs

```
[POST] /api/picker?initialPath=%252Fapi&previousButtonValues=%2523A_A%252CB&nxtProutes=picker status=500
```

```
Error: Invalid frame url: https://meme-frame-frog.vercel.app/api. Expected: https://meme-frame-frog.vercel.app/api/picker?initialPath=%252Fapi&previousButtonValues=%2523A_A%252CB.
```

If I remove the hub, then it works fine and the `/api/picker` endpoint doesn't error.

**Comment by codexbushi** [comment, marked as answer]
Seems like this has been fixed with a recent release. I updated my frog to `0.3.3` and used the `pinata()` hub.
